server/src/main/java/org/activityinfo/server/endpoint/akvo/flow/QuestionAnswer.java

```diff
@@ -0,0 +1,24 @@
+package org.activityinfo.server.endpoint.akvo.flow;
+
+final public class QuestionAnswer {
+    static final public class Array {
+        public QuestionAnswer question_answers[];
+    }
+
+    static final public class Single {
+        public QuestionAnswer question_answer;
+    }
+
+    public Object value;
+    public String type;
+    public Object oldValue;
+    public Integer arbitratyNumber;     // Sic!
+    public Integer surveyId;
+    public String questionId;
+    public String questionText;
+    public Long collectionDate;
+    public Integer surveyInstanceId;
+    public Integer keyId;
+    public String textualQuestionId;
+    public String questionID;           // Presumably a miscapitalization of questionId
```

Comment:
Oh, this is a part of the code I was going to refactor anyway... The only reason this is a pull request is that it allows me to ask for rebuilds. I never meant to imply that this is pull-worthy.